includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -467,13 +491,14 @@ public function generate_payment_request( $order, $prepared_source ) {
 		 * @param WC_Order $order
 		 * @param object $source
 		 */
-		return apply_filters( 'wc_stripe_generate_payment_request', $post_data, $order, $prepared_source );
+		return apply_filters( 'wc_stripe_generate_payment_request', $post_data, $order, $payment_method );
 	}
 
 	/**
 	 * Store extra meta data for an order from a Stripe Response.
 	 */
 	public function process_response( $response, $order ) {
+		// TODO: This does not support setup intents.
```

Comment:
Does this TODO apply to using saved payments methods on pay for order page?

Reply:
I left this `TODO` thinking more about the case of a free-trial subscription, where we would complete a checkout using a setup intent, but would probably still want to update the order status using `process_response`. I _think_ you would be able to do this from the Pay for Order page as well, however, this `TODO` applies to any checkout using a free-trial subscription and not just those using saved payment methods.